modules/virt-hot-plugging-memory.adoc

```diff
@@ -20,5 +20,7 @@ The system applies these changes immediately. If the VM is migratable, a live mi
 
 [NOTE]
 ====
-Linux guests require a kernel version of 5.16 or later and Windows guests require the latest `viomem` drivers.
-====
\ No newline at end of file
+* Linux guests require a kernel version of 5.16 or later and Windows guests require the latest `viomem` drivers.
+* By default, a virtual machine (VM) can be hot-plugged with up to four times its initially assigned CPU count. For example, a VM with 1 vCPU can scale up to 4 vCPUs before a restart is required. If the VM is migratable, the change is applied through live migration. If not, or if the change cannot be live-updated, a restart is required.

```

Comment:
```suggestion
* By default, a VM can be hot-plugged with up to four times its currently assigned CPU count. For example, a VM with 1 vCPU can scale up to 4 vCPUs before a restart is required. If the VM is can be migrated, the change is applied through live migration. If not, or if the change cannot be live-updated, a restart is required.

```
OK, so if I modify the limit, then that is applied per the modified assignment, is that correct? If yes, then I think we should say "currently" not "initially" in the first line of the topic, because we are starting with the live state of the VM; not the state it was when we first created it, which would be the "initial" state.